tests/shared.py

```diff
@@ -36,6 +37,9 @@ def __init__(self, model):
         self.rank = 0
         self.local_rank = 0
         self.log_every_n_steps = 1e8
+        self.save_logs = True
```

Comment:
maybe keep `save_logs False` by default and we can set it in just the tests where we care about logs?

Reply:
If set to false my understanding is that it doesn't save the checkpoint at all. BTW is that a bug?

Reply:
i see okay, that is weird. https://github.com/mlfoundations/open_lm/blob/73e7b3709057d62408d700dcaad4ebc6317c56b9/open_lm/main.py#L610 should set save_logs to True if you've specified a logs dir..

Reply:
I set it to false in the shared file but I did have to set it to true in my test otherwise the fsdp test would not pass (not sure why).